cpu/stm32l1/periph/i2c.c

```diff
@@ -34,17 +34,14 @@
 #define ENABLE_DEBUG    (0)
 #include "debug.h"
 
-/* guard file in case no I2C device is defined */
-#if I2C_NUMOF
-
```

Comment:
@PeterKietzmann I though the drivers are used now with "best knowledge", that the person who use them, knows how his board works? Or is it different with I2C ?